notebooks/remote_functions/remote_function_vertex_claude_model.ipynb

```diff
@@ -0,0 +1,484 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Use BigQuery DataFrames to run Anthropic LLM at scale\n",
+    "\n",
+    "<table align=\"left\">\n",
+    "\n",
+    "  <td>\n",
+    "    <a href=\"https://colab.sandbox.google.com/github/googleapis/python-bigquery-dataframes/blob/main/notebooks/remote_functions/remote_function_vertex_claude_model.ipynb\">\n",
```

Comment:
Let's keep it colab.sandbox (also below), colab.research redirects to colab.sandbox internally but colab.sandbox itself isn't accessible publically

Reply:
Done